Armor.java

```diff
@@ -1,28 +1,32 @@
 import java.util.Random;
-public class Armor{
+public class Armor {
     private String name;
     private double dmgAbs;
-    private int aHealth;
+    private double aHealth;
```

Comment:
```suggestion
    private double aHealth;
    private double aMaxHealth;
```
for when we make the armor reparable